### DISCUSSION TITLE
Welcome to comp3340Project Discussions!

### DISCUSSION THREAD

**Discussion by arun-esh** [arun-esh/travelx#58] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jalpa015** [comment]
Hello, 
I am Jalpa Patel. It is a great project that you have been working on.
I am an experienced Software Developer (Backend, Mobile and Web Apps). Currently picking up and mastering in ML and AI skills. I love the concept of community engagement besides technical work and projects. I have been trying to actively engage in community to share my knowledge and also learn from other's experiences. Currently I am engaging with Women Who Code community that encourages women developers and provides with various resources.
